app/assets/javascripts/host_edit_interfaces.js

```diff
@@ -276,6 +284,15 @@ $(document).on('click', '.provision-flag', function () {
   update_interface_table();
 });
 
+$(document).on('click', '.managed-flag', function onClickManaged() {
```

Comment:
Can we use a single function for all three: primary, managed and provisioning with the interface-ids being updated inside if-else ?